packages/primitives/src/Avatar/Avatar.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react'
+import { Image } from 'theme-ui'
+import { SxStyleProp } from '@kodiak-ui/core'
+import { Box } from '../Box'
+
+export type AvatarProps = {
+  imageSrc?: string
+  text: string
```

Comment:
@seri, we shouldn't need this either because we should just be able to pass `alt` to `<Avatar alt="text" />` to add the alt text to the image.